python/src/phylanx.cpp

```diff
@@ -321,6 +325,26 @@ PYBIND11_MODULE(_phylanx, m)
             &phylanx::bindings::pickle_helper<phylanx::ast::expression>,
             &phylanx::bindings::unpickle_helper<phylanx::ast::expression>));
 
+    // list of phylanx::ast::operations
+    pybind11::class_<std::list<phylanx::ast::operation>>(ast,"operation_list","A list of operations")
+        .def(pybind11::init<>())
+        .def("pop_back",&std::list<phylanx::ast::operation>::clear)
+        .def("append",[](std::list<phylanx::ast::operation>& v,const phylanx::ast::operation& f) { v.push_back(f); })
+        .def("__len__",[](const std::list<phylanx::ast::operation>& v) { return v.size(); })
+        .def("__iter__",[](std::list<phylanx::ast::operation>& v) {
+            return pybind11::make_iterator(v.begin(), v.end());
+        }, pybind11::keep_alive<0,1>());
+
+    // list of phylanx::ast::expressions
+    pybind11::class_<std::list<phylanx::ast::expression>>(ast,"expression_list","A list of expressions")
+        .def(pybind11::init<>())
+        .def("pop_back",&std::list<phylanx::ast::expression>::clear)
```

Comment:
This should probably call `std::list<>::pop_back()`, not `std::list<>::clear()`.

Reply:
I fixed this.